src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
+
+import QuizContext from '../CioQuiz/context';
+
+export default function ProgressBar() {
+  const { state } = useContext(QuizContext);
+  const currentQuestion = state?.quiz.currentQuestion;
+  if (!currentQuestion) return null;
+
+  return (
+    <div className='cio-question-progress-affixed-container'>
+      <div className='cio-question-progress-progress-container'>
+        <div
+          className='cio-question-progress-progress-bar'
+          style={{
+            width: `${
+              ((currentQuestion.next_question.id - 1) / currentQuestion.total_questions) * 100
```

Comment:
@lws803 from Quiz Service perspective, can we always depend on `id` to be equal to `index`?